2020-01-06 22:53:42,747 ERROR: Exception on /index [GET] [in c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py:1780]
Traceback (most recent call last):
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\_compat.py", line 36, in reraise
    raise value
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\40382\Desktop\研一上课程\flask_blog\app\routes.py", line 53, in index
    prev_url=prev_url)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\40382\Desktop\研一上课程\flask_blog\app\templates\index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\40382\Desktop\研一上课程\flask_blog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block "body"
    {% block content -%}
  File "C:\Users\40382\Desktop\研一上课程\flask_blog\app\templates\base.html", line 48, in block "content"
    {% block app_content %}
  File "C:\Users\40382\Desktop\研一上课程\flask_blog\app\templates\index.html", line 7, in block "app_content"
    {{ wtf.quick_form(form) }}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\wtf.html", line 205, in template
    {{ form_field(field,
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\jinja2\runtime.py", line 579, in _invoke
    rv = self._func(*arguments)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\flask_bootstrap\templates\bootstrap\wtf.html", line 123, in template
    {{field(class="form-control", **kwargs)|safe}}
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\wtforms\fields\core.py", line 155, in __call__
    return self.meta.render_field(self, kwargs)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\wtforms\meta.py", line 56, in render_field
    return field.widget(field, **render_kw)
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\wtforms\widgets\core.py", line 323, in __call__
    for val, label, selected in field.iter_choices():
  File "c:\users\40382\appdata\local\programs\python\python37\lib\site-packages\wtforms\fields\core.py", line 455, in iter_choices
    yield (value, label, self.coerce(value) == self.data)
ValueError: invalid literal for int() with base 10: '电视机开12:30，电视机关15:30'
2020-01-06 22:54:07,667 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-06 22:57:25,479 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-06 23:00:52,122 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-06 23:00:57,070 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-09 00:10:52,172 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-15 08:01:39,334 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 00:45:23,139 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 10:38:56,793 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 10:39:18,730 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 11:03:53,863 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 11:06:26,290 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 12:32:10,552 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 12:50:33,285 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 15:49:47,071 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
2020-01-16 15:52:22,310 INFO: Microblog startup [in C:\Users\40382\Desktop\研一上课程\flask_blog\app\__init__.py:56]
